src/index.ts

```diff
@@ -1,25 +1,40 @@
-import { readFileSync } from 'fs';
+import _chainsDeploymentInfo from '@api3dao/operations/data/chains.json';
+import { goSync, isGoSuccess } from './utils';
 
-function getServiceData(serviceName: string, chain: string) {
-  const deploymentJson = JSON.parse(readFileSync(`./deployments/${chain}.json`).toString());
-  if (!deploymentJson) {
+interface ChainInfo {
+  AirnodeRrp: string;
+  AccessControlRegistry: string;
+  OwnableCallForwarder: string;
+  RequesterAuthorizerWithManager: string;
+  RrpBeaconServer: string;
+}
+
+// Typing this more loosly to avoid TS errors
+const chainsDeploymentInfo: { [chainName: string]: ChainInfo } = _chainsDeploymentInfo;
+
+function getServiceData(apiName: string, beaconName: string, chain: string) {
+  const chainInfo = chainsDeploymentInfo[chain];
+  if (!chainInfo) {
     throw new Error(`Deployment file for ${chain} not found`);
   }
 
-  // TODO: import from @operations and filter by serviceName param
-  const ethToUsdJson = JSON.parse(readFileSync('../opeartions/data/apis/Amberdata/beacons/ethToUsd.json').toString());
+  const goBeaconData = goSync(() => require(`@api3dao/operations/data/apis/${apiName}/beacons/${beaconName}`));
+  if (!isGoSuccess(goBeaconData)) throw new Error(`Service data does not exist.`);
+  const beaconData = JSON.parse(goBeaconData.data).toString();
 
+  // TODO: Create an TS interface type for this
+  // TODO: Should we return the whole beaconData?
   return {
```

Comment:
I think we now need to review this return object since previously it was returning information about a service in a chain (e.g. Amberdata) and now it seems that by adding the `beaconName` as an arg then we should be returning beacon data in a specific chain.

Reply:
beacon-reader-example only requires `RrpBeaconServer` contract address and `beaconId` but I initially also added `AirnodeRrp` and other data because I wanted to make the return object more useful to anyone calling this function.

Reply:
I actually just parametrized the beacon name (e.g. previously `ethToUsd.json`), but I agree that what we retutrn should be thought through.

As mentioned on slack - I think the best would be to return only data needed for the beacon = rrpBeaconAddress and beacon id.

in the future (but it should be before hacathon), we should think about what exactly to return. I'd say we can return a lot more (e.g. the full beaconData), but we should go through the fields and their naming.